br/pkg/restore/split/split.go

```diff
@@ -174,27 +175,45 @@ func ScanRegionsWithRetry(
 		}
 
 		return nil
-	}, newScanRegionBackoffer())
+	}, NewScanRegionBackoffer())
 
 	return regions, err
 }
 
 type scanRegionBackoffer struct {
-	attempt int
+	attempt      int
+	delayTime    time.Duration
+	maxDelayTime time.Duration
 }
 
-func newScanRegionBackoffer() utils.Backoffer {
+// NewScanRegionBackoffer create a backoff to retry to scan regions.
+func NewScanRegionBackoffer() utils.Backoffer {
 	return &scanRegionBackoffer{
-		attempt: ScanRegionAttemptTimes,
+		attempt:      ScanRegionAttemptTimes,
+		delayTime:    time.Millisecond * 10,
+		maxDelayTime: time.Second * 2,
 	}
 }
 
 // NextBackoff returns a duration to wait before retrying again
 func (b *scanRegionBackoffer) NextBackoff(err error) time.Duration {
 	if berrors.ErrPDBatchScanRegion.Equal(err) {
-		// 1s * 60 could be enough for splitting remain regions in the hole.
+		// it needs more time to wait splitting the regions that contains data in PITR.
+		// 2s * 150
 		b.attempt--
-		return time.Second
+		b.delayTime *= 2
```

Comment:
You can embed `utils.RetryState` if you need to implement the Exponential Backoff. Just like 
```go
if berrors.ErrPDBatchScanRegion.Equal(err) {
  delayTime := state.ExponentialBackoff()
  failpoint.Inject(...)
  return delayTime
}
```